### DISCUSSION TITLE
webdav as backend

### DISCUSSION THREAD

**Discussion by diseq** [juicedata/juicefs#566] [Q&A] [answered]
Hi!

I saw there is an implementation of a restful client (restful.go)
However, this is used as part of other backends.

What is the proposed way to use a generic webdav backend?
As i understood restful.go cannot be used directly.

**Comment by xiaogaozi** [comment, marked as answer]
FYI, we already created an issue (https://github.com/juicedata/juicefs/issues/568), it will be supported in the future.

**Reply by diseq** [reply]
awesome! thanks

**Reply by davies** [reply]
It's supported in latest main branch, will be release in next week.

FYI: https://github.com/juicedata/juicefs/blob/main/docs/en/how_to_setup_object_storage.md#webdav
